### PR TITLE
Logging to debug fancode timeout errors

### PR DESCRIPTION
**Pull request by naseem-shaik**
## JIRA Ticket Number

JIRA TICKET: ML-3287

## Description of change
Added logs in transfer data in chatbot_ner for debugging

### CODE REVIEW

File: datastore/elastic_search/transfer.py [hunk 1]
```diff
@@ -378,8 +387,12 @@ def transfer_data_internal(es_url, index_to_backup, backup_index):
                 "index": backup_index
             }
         }
+
+        ner_logger.debug('Start post request made with es_url reindex'
+                            'es_url: %s' % (es_url))
```
Comment (haptik-deployment) [inline datastore/elastic_search/transfer.py:392]: E127 continuation line over-indented for visual indent

**Review by harjinder7 — DISMISSED** [review]
Code changes looks fine to me.
Linting feedback need to be fix

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Comment by haptik-deployment** [comment]
:+1: No lint errors found.

**Review by harjinder7 — APPROVED** [review]

**Review by HiteshC6595 — APPROVED** [review]

**Pull request merged**